DevCycle/Models/DVCEvent.swift

```diff
@@ -100,4 +100,15 @@ struct DVCAggregateEvents {
             }
         }
     }
+    
+    func getAllAggregateEvents() -> [DVCEvent] {
```

Comment:
maybe this should be a seperate ticket, but we should really be storing this data in a two layer map like: `[String: [String:DVCEvent]]`, where the first key is type like `variableDefaulted`, we shouldn't have to add a map like this every time we add an aggregated event type.

Reply:
we can create it as a todo

Reply:
k, can you create a ticket for that?

Reply:
yep

Reply:
done DVC-2185